samples/compiler_plugins/xnnpack/src/xnnpack/Conversion/StablehloToXnnpack.cpp

```diff
@@ -272,10 +269,22 @@ class ConvertFullyConnectedLayer
       return error("not a fully connected pattern");
     }
 
+    Type expectedInputElemType = IntegerType::get(op.getContext(), /*width=*/8);
```

Comment:
rename it by `i8Ty` to be more specific. Same for `i4Ty` and `f32Ty` below.

Reply:
Done!